Modify S5147(Common text): Update to the education framework (APPSEC-94) (#1195)
SonarSource/rspec: rules/S5147/rationale.adoc

> NoSQL injections occur in an application when the application retrieves data
> from a user or a third-party service and inserts it into a database query
> without sanitizing it first.

Reviewer: ```suggestion
NoSQL injections occur when an application retrieves untrusted data and inserts
it into a database query without sanitizing it first.
```
